server/index.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const path = require('path');
+
+const app = express();
+
+const port = 3000;
+
+app.use(express.static(path.join(__dirname, '../public')));
+
+app.listen(port, () => console.log('[Server] Listening on port 3000'));
```

Comment:
add a newline here

Reply:
Good catch! I totally forgot to set up ESLint here, going to do that next